request/mandatory_parameters.go

```diff
@@ -0,0 +1,18 @@
+package request
+
+import "net/http"
+
+//ErrorMissingRequiredParameters is returned when a request does not contain required parameters
+var ErrorMissingRequiredParameters error
```

Comment:
The Error is for clients using this library to return, but I suppose it could be returned here too